src/main/java/edu/escuelaing/ieti/matchmaking/exception/InsufficientFundsException.java

```diff
@@ -0,0 +1,14 @@
+package edu.escuelaing.ieti.matchmaking.exception;
+
+import org.springframework.util.StringUtils;
+
+public class InsufficientFundsException extends Exception {
```

Comment:
Extend this exception from `RuntimeException` so that you can avoid putting it in the throws clause of every single method